### PR TITLE
Add resolve_age as optional project parameter

### PR DESCRIPTION
**Pull request by mced**
Signed-off-by: Cédric Menassa <cedric.menassa@gmail.com>

### CODE REVIEW

**Comment by mced** [comment]
@jianyuan any chance to see this pr merged ?

File: sentry/resource_sentry_project.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func resourceSentryProject() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"resolve_age": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
```
Comment (jianyuan) [inline sentry/resource_sentry_project.go]: Can we add Optional: true and a description?

```suggestion
			"resolve_age": {
				Type:        schema.TypeInt,
				Optional:    true
				Description: "Hours in which an issue is automatically resolve if not seen after this amount of time.",
				Computed:    true,
			},
```

**Comment by mced** [comment]
@jianyuan resolved

**Comment by jianyuan** [comment]
Looks good to me, thank you!

**Pull request merged**